src/amo/sagas/collections.js

```diff
@@ -164,17 +161,19 @@ export function* addAddonToCollection({
     };
     yield call(api.createCollectionAddon, params);
 
-    yield put(addonAddedToCollection({
-      addonId, userId, collectionId,
-    }));
-
     if (editing) {
-      yield put(fetchCurrentCollectionAction({
-        fetchAllAddons: true,
+      invariant(page, 'A page parameter is required when editing');
```

Comment:
This is needed for Flow, because `addAddonToCollection` can legally be dispatched without a `page` parameter, when displatched from the `AddAddonToCollection` component, but it's needed when dispatching from the `CollectionManager` component.

Reply:
ah ok.. let's keep it then.